posthog/async_migrations/migrations/0005_person_replacing_by_version.py

```diff
@@ -70,6 +70,9 @@ class Migration(AsyncMigrationDefinition):
 
     depends_on = "0004_replicated_schema"
 
+    posthog_min_version = "1.38.0"
+    posthog_max_version = "1.39.99"
```

Comment:
Why? There isn't a reason to make this mandatory in the next release _yet_.

I'm against this.

Reply:
discussed in sync